### ISSUE TITLE
Separate audio tracks for Twitch

### ISSUE THREAD

**Issue by tommybounce** [fzwoch/obs-teleport#55]
Twitch DMCA is a pain in the *** nowadays. 
I havent been able to separate audio tracks through Teleport so that music wont be added in VOD.

**Comment by fzwoch** [comment]
That is currently not supported. I don't have any plans myself for adding that feature at the moment.

**Comment by no-connections** [comment]
Can't you just send another audio only stream from whatever source you have for audio?

Multi track audio in Teleport could be useful, but there are not that many uses for it.

**Comment by PipeL1n3** [comment]
Just seperate the tracks in sending OBS. Don´t use the audio from main source, but instead using teleport on each audio source you have in sending obs. Add them to scene you want to and toggle the audio track you want them to be in on your receiving obs. mute the audio from main-teleport.

Its possible to do it that way. But @fzwoch it would be an nice option to have teleport-main-output as a seperate audio track in obs, so you can toggle on/off what audio sources will be sent via teleport. This would be nice to have feature.

**Comment by fzwoch** [comment]
Closing. More complex audio setups should be done before reaching Teleport.

**Comment by copygirl** [comment]
I just want to throw my two cents into the mix here.

I'm moving to having my gaming and streaming to be done on different machines. I had been using the [Waveform](https://github.com/phandasm/waveform) plugin to display a visualizer for the music being played on stream. If I wanted to move more functionality to my streaming PC, I would not be able to use the plugin there, since the audio streams are merged and I can't differentiate between music, game audio and my mic any longer.

Of course, I could just run the plugin on my gaming PC. I suppose this is where I need to add some context: My "streaming PC" is actually a Steam Deck which I've been abusing to take some load of my (slightly older) desktop PC, especially with running some VTuber tracking software and such. Recently I've even set up my (much weaker) laptop to push video to my Steam Deck via Teleport, and that worked pretty okay. In this situation it would be very nice to de-duplicate the setup so no matter where I am, I have all the streaming related functionality on my Steam Deck (the streaming PC).

Teleport supporting separate audio tracks would be nice.

**Comment by fzwoch** [comment]
You can still use Teleport on any audio source and transmit audio as a separate stream?

**Comment by copygirl** [comment]
I didn't know that was a thing! I'll definitely check that out!
